repo_resource/common.py

```diff
@@ -240,11 +241,17 @@ def __change_to_workdir(self):
     def __restore_oldpwd(self):
         os.chdir(self.__oldpwd)
 
-    def __add_remote(self, remote, url):
-        self.__remote[remote] = url
+    def __add_remote_url(self, remote, url):
+        self.__rmt_url[remote] = url
```

Comment:
Please, keep member named `__remote_url`. `rmt` is a non-standard shorthand, making things more difficult to understand.

Reply:
Python does not handle __remote_url function (declared just below) and __remote_url dict, it always calls the dict if they're called the same and throws an error
I can change the function to __get_remote_url. I'm open to other suggestions

Reply:
Ah I see. The problem is that the methods and the member have the same name.

I'm ok with `__add_remote_url()` and `__get_remote_url()`